website/assets/js/cloud.setup.js

```diff
@@ -13,7 +13,7 @@
 Cloud.setup({
 
   /* eslint-disable */
```

Comment:
Reminder: Be sure to regenerate this file using `sails run rebuild-cloud-sdk` after any changes to the API endpoints in `config/routes.js` (like the one requested in this PR review)